diff --git a/src/build_tools/mozc_win32_resource_template.rc b/src/build_tools/mozc_win32_resource_template.rc
index deec83422..7f12ed07e 100644
--- a/src/build_tools/mozc_win32_resource_template.rc
+++ b/src/build_tools/mozc_win32_resource_template.rc
@@ -6,7 +6,7 @@
   #if defined(GOOGLE_JAPANESE_INPUT_BUILD)
     #define MOZC_RES_COMPANY_NAME "Google LLC"
   #elif defined(MOZC_BUILD)
-    #define MOZC_RES_COMPANY_NAME "Google LLC"
+    #define MOZC_RES_COMPANY_NAME "Mozc Project, MATANUKI"
   #endif
 #endif
 #ifndef MOZC_RES_FILE_DESCRIPTION
@@ -16,7 +16,7 @@
   #define MOZC_RES_INTERNAL_NAME ""
 #endif
 #ifndef MOZC_RES_LEGAL_COPYRIGHT
-  #define MOZC_RES_LEGAL_COPYRIGHT "Copyright 2026 Google LLC. All Rights Reserved."
+  #define MOZC_RES_LEGAL_COPYRIGHT "Copyright 2026 Mozc Project, MATANUKI. All Rights Reserved."
 #endif
 #ifndef MOZC_RES_ORIGINAL_FILENAME
   #define MOZC_RES_ORIGINAL_FILENAME ""
@@ -27,5 +27,5 @@
   #elif defined(MOZC_BUILD)
-    #define MOZC_RES_PRODUCT_NAME "Mozc"
+    #define MOZC_RES_PRODUCT_NAME "Mozc4med"
   #endif
 #endif
 #ifndef MOZC_RES_FILE_VERSION_NUMBER
