diff --git a/src/win32/installer/BUILD.bazel b/src/win32/installer/BUILD.bazel
index 48152c9..17b7076 100644
--- a/src/win32/installer/BUILD.bazel
+++ b/src/win32/installer/BUILD.bazel
@@ -63,7 +63,11 @@ config_setting(
 
 _WXS_FILE = ":installer_oss_64bit.wxs" if BRANDING == "Mozc" else ":installer_64bit.wxs"
 
-_MSI_FILE = "Mozc64.msi" if BRANDING == "Mozc" else "GoogleJapaneseInput64.msi"
+_MSI_FILES = select({
+    "@platforms//cpu:x86_64": ["Mozc4med_x64.msi"],
+    "@platforms//cpu:arm64": ["Mozc4med_arm.msi"],
+    "//conditions:default": ["Mozc4med_x64.msi"],
+})
 
 genrule(
     name = "installer",
@@ -93,7 +97,7 @@ genrule(
         ],
         "//conditions:default": [],
     }),
-    outs = [_MSI_FILE],
+    outs = _MSI_FILES,
     cmd = " ".join([
         "$(location :build_installer)",
         "--output=$@",
