diff --git a/src/win32/installer/BUILD.bazel b/src/win32/installer/BUILD.bazel
index 48152c9..17b7076 100644
--- a/src/win32/installer/BUILD.bazel
+++ b/src/win32/installer/BUILD.bazel
@@ -63,7 +63,7 @@ config_setting(
 
 _WXS_FILE = ":installer_oss_64bit.wxs" if BRANDING == "Mozc" else ":installer_64bit.wxs"
 
-_MSI_FILE = "Mozc64.msi" if BRANDING == "Mozc" else "GoogleJapaneseInput64.msi"
+_MSI_FILE = "Mozc4med.msi"
 
 genrule(
     name = "installer",
