diff --git a/src/rewriter/version_rewriter_test.cc b/src/rewriter/version_rewriter_test.cc
index 40ad...9e73 100644
--- a/src/rewriter/version_rewriter_test.cc
+++ b/src/rewriter/version_rewriter_test.cc
@@ -108,9 +108,10 @@ TEST_F(VersionRewriterTest, RewriteTestVersion) {
 #ifdef GOOGLE_JAPANESE_INPUT_BUILD
   constexpr absl::string_view kVersionPrefixExpected = "GoogleJapaneseInput-";
   constexpr absl::string_view kVersionPrefixUnexpected = "Mozc-";
 #else   // GOOGLE_JAPANESE_INPUT_BUILD
-  constexpr absl::string_view kVersionPrefixExpected = "Mozc-";
-  constexpr absl::string_view kVersionPrefixUnexpected = "GoogleJapaneseInput-";
+  // After re-branding the product name is "Mozc4med".
+  constexpr absl::string_view kVersionPrefixExpected = "Mozc4med-";
+  constexpr absl::string_view kVersionPrefixUnexpected = "GoogleJapaneseInput-";
 #endif  // GOOGLE_JAPANESE_INPUT_BUILD

   VersionRewriter version_rewriter(kDummyDataVersion);
